src/migrations/20250805152422-backfill-lifecycle-trends.js

```diff
@@ -0,0 +1,91 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+            DELETE FROM lifecycle_trends;
+
+            WITH week_ranges AS (
+                SELECT
+                    to_char(week_start, 'YYYY') || '-' || lpad(extract(week from week_start)::int::text, 2, '0') AS id,
```

Comment:
Is the `0` in here what makes this a sunday-based thing?

Reply:
Ha, actually no. It is for week numbers, that are 1 digit like 2025-9, so it pads it with 0, making it 2025-09.